sign/ed25519/signapi.go

```diff
@@ -0,0 +1,75 @@
+package ed25519
```

Comment:
Nearly identical to the other `signapi.go` files (I know there are some differences). Is it generated? If so it needs a generated comment.

Reply:
No, it's not.